src/auro-badge.js

```diff
@@ -48,8 +66,12 @@ class AuroBadge extends LitElement {
   // function that renders the HTML and CSS into  the scope of the component
   render() {
     return html`
-      <div class=${this.cssClass}>
+      <div class="badge" @click=${this.handleChange} role="button">
```

Comment:
This should be a `<button>`. A `<div>` is not able to be focused or pressed by the keyboard. `role="button"` tells assistive tech that this item is a button control, but does not make it focusable or add keyboard handlers (should be interactive on space/enter). See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Once we make it focusable, we also need to add a visual `:focus` state.

Reply:
@erikalanghauser - would it be possible to get a mockup for focus-visible state here?

Reply:
@geoffrich would you recommend making _just_ the 'X' action a focus-visible entity (button)? or would you recommend making the entire badge an icon? it seems like Shoelace follows the former pattern - https://shoelace.style/components/tag

Reply:
Good point -- I think I'd follow what shoelace is doing. It doesn't make sense to make the whole component focusable if only the close button is able to be interacted with. So maybe conditionally render a `<button>closeSvg</button>` and attach the click handler to that?

Reply:
@gusnaughton lmk if you want a clean mockup for the focus state around the 'X' or if you have what you need from this thread and previous focus state work. Thanks!

Reply:
@erikalanghauser I think I have what I need - I'll take a leaf out of auro-button's book.

Reply:
@gusnaughton -- thanks for adding a `button` element! I'm still not seeing a visible focus state when I tab with the keyboard, though.